01week/datatypes.js

```diff
@@ -0,0 +1,73 @@
+// Write a JavaScript program to display the current day and time.
+const displayDateAndTime = () =>{
+  //Display day  
+    const todaysName = newDate();
+    const today = todaysName.getDay();
+  //Display time
+    const t = newTime();
+    const time = t.getTime();  
+  } 
```

Comment:
Fix the indentation for this curly brace, it should line up with it's opening brace (there should be no spaces before it)